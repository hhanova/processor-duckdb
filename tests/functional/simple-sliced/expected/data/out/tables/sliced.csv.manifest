{"destination": "sliced", "primary_key": ["session_id"], "columns": ["track", "session_id", "session_name", "new_column"], "incremental": true, "enclosure": "\"", "delimiter": ",", "metadata": [], "column_metadata": {"track": [{"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": false}], "session_id": [{"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": false}], "session_name": [{"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": false}], "new_column": [{"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}]}, "write_always": false}