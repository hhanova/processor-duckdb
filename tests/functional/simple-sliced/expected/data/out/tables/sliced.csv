"product_owner","1152674","Platform Introduction (PR)","1"
"data_specialist","1152674","Platform Introduction (PR)","1"
"data_specialist","1249022","SQL Syntax Training (PR)","1"
"data_specialist","1152675","Best Practices (PR)","1"
"data_specialist","1152676","Debug Techniques (PR)","1"
"data_specialist","1155656","Common Components and Processors (PR)","1"
"sr_data_developer","1152674","Platform Introduction (PR)","1"
"sr_data_developer","1249022","SQL Syntax Training (PR)","1"
"sr_data_developer","1152675","Best Practices (PR)","1"
"sr_data_developer","1152676","Debug Techniques (PR)","1"
"sr_data_developer","1174202","Generic Extractor (PR)","1"
"sr_data_developer","1155656","Common Components and Processors (PR)","1"
"sr_data_developer","1174274","KBC APIs (PR)","1"
"sr_data_developer","1173117","Data Science (PR)","1"
"platform_developer","1152674","Platform Introduction (PR)","1"
"platform_developer","1249022","SQL Syntax Training (PR)","1"
"platform_developer","1152675","Best Practices (PR)","1"
"platform_developer","1152676","Debug Techniques (PR)","1"
"platform_developer","1174202","Generic Extractor (PR)","1"
"platform_developer","1174222","Project Health Check (PR)","1"
"platform_developer","1155656","Common Components and Processors (PR)","1"
"platform_developer","1174224","Component Behind the Scenes (PR)","1"
"platform_developer","1174264","Component Creation (PR)","1"
"platform_developer","1174274","KBC APIs (PR)","1"
"platform_developer","1173117","Data Science (PR)","1"
"platform_developer","1173118","Data Governance (PR)","1"
"data_scientist","1152674","Platform Introduction (PR)","1"
"data_scientist","1249022","SQL Syntax Training (PR)","1"
"data_scientist","1152675","Best Practices (PR)","1"
"data_scientist","1152676","Debug Techniques (PR)","1"
"data_scientist","1155656","Common Components and Processors (PR)","1"
"data_scientist","1173117","Data Science (PR)","1"
"admin","1152674","Platform Introduction (PR)","1"
"admin","1249022","SQL Syntax Training (PR)","1"
"admin","1152675","Best Practices (PR)","1"
"admin","1152676","Debug Techniques (PR)","1"
"admin","1174202","Generic Extractor (PR)","1"
"admin","1174222","Project Health Check (PR)","1"
"admin","1155656","Common Components and Processors (PR)","1"
"admin","1174274","KBC APIs (PR)","1"
"admin","1173117","Data Science (PR)","1"
"admin","1173118","Data Governance (PR)","1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
,"#N/A",,"1"
